_posts/2021-02-09-open-sdg-1-2-0.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 1.2.0 released
+excerpt: Open SDG 1.2.0 is now available! Here are the details.
+author_profile: false
+---
+Open SDG, a reporting platform developed for managing and publishing data and statistics related to the UN Sustainable Development Goals (SDGs) is built with open-source libraries and tools. It is freely available for any country or organization to reuse and can be hosted and maintained using free services. Last month, the Open SDG community launched version 1.2.0, marking a significant milestone in the Open SDG journey. 1.2.0 includes a number of design upgrades and notable new features.
+
+More is in the works, with version 1.3.0 already in development along with new resources - including a series of training videos - that will make it easier for new governments to adopt Open SDG for their own needs.
+
+The key updates included in version 1.2.0 focus on making the site more accessible. Those upgrades include more accessible charts, updated hierarchy for screen readers, accessibility fixes and improvements for sorting tables and chart legends, and a number of changes to Open-SDG’s mapping functions. Other notable updates include translated page titles and content, custom source-code file downloads, and a number of design and style upgrades. You can view a summary of the updates on our [platform updates page](https://open-sdg.readthedocs.io/en/latest/updates/), and a more detailed list in our [1.2.0 release notes](https://github.com/open-sdg/open-sdg/releases/tag/1.2.0).
```

Comment:
I'm wondering if this paragraph could be:

The key updates included in version 1.2.0 focus on making the site more accessible, including:
- more accessible charts
- updated hierarchy for screen readers
- accessibility fixes _**I think this could actually be removed as general accessibility fixes is already mentioned in leading sentence**_
- improvements for sorting tables and chart legends
- a number of changes to Open-SDG’s mapping functions.

Other notable updates include:
- translated page titles and content
- custom source-code file downloads
- a number of design and style upgrades

You can view a summary of the updates on our [platform updates page](https://open-sdg.readthedocs.io/en/latest/updates/), and a more detailed list in our [1.2.0 release notes](https://github.com/open-sdg/open-sdg/releases/tag/1.2.0).